### PR TITLE
Adding Norwegain bokmål extension

### PR DESCRIPTION
**Pull request by spiftire**

### CODE REVIEW

File: extensions/norwegian-bokmal/src/extension.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+// Import the module and reference it with the alias vscode in your code below
+import * as vscode from 'vscode';
+interface CodeSpellCheckerExtension {
+    registerConfig(path: string): Promise<void>;
+    enableLocal(isGlobal: boolean, locale: string): Promise<void>;
+    disableLocal(isGlobal: boolean, locale: string): Promise<void>;
+}
+
+
+const locale = 'nb';
```
Comment (Jason3S) [inline extensions/norwegian-bokmal/src/extension.ts:12]: After seeing this, I realize we got the name of the dictionary wrong. I'll change it.
The dictionary should be Language-Country, so no_NB should be nb_NO.

Comment (Jason3S) [reply extensions/norwegian-bokmal/src/extension.ts:12]: I have fixed the dictionary on NPM.

Please change any references like `no_NB` or `no-nb` to `nb_NO` ans `nb-no` respectively.

Please add a `cspell.json` file in the extension directory with `"words": ["Bokmål"]` to fix the spelling complaints.

Please note: the dependency in `package.json` needs to be  `@cspell/dict-nb-no`

Comment (Jason3S) [reply extensions/norwegian-bokmal/src/extension.ts:12]: Or add them to the `cusotm-words.txt` like you did.

Comment (spiftire) [reply extensions/norwegian-bokmal/src/extension.ts:12]: Done and done. The pipeline jobs should pass now

**Comment by spiftire** [comment]
No problemo. I love the work you have done here. Just happy to help out 🚀

**Comment by Jason3S** [comment]
@spiftire [Norwegian Bokmål - Code Spell Checker - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker-norwegian-bokmal)

**Pull request merged**